src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,534 @@
+          recordsInCuration: [],
+          recordsWithoutDois: [],
+          hiddenRecords: [],
+          recordType: null,
+          headers: {
+            approvalRequired: [
+              {
+                text: "Date last modification",
+                value: "updatedAt"
+              },
+              {
+                text: "Curator",
+                value: "curator"
+              },
+              {
+                text: "Record name (id)",
+                value: "recordName"
+              },
+              {
+                text: "Last editor",
+                value: "lastEditor"
+              }
+            ],
+            maintenanceRequests: [
+              {
+                text: "Date",
+                value: "createdAt"
+              },
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "User login (id)",
+                value: "userNameID"
+              }
+            ],
+            recordsCreatedCuratorsLastWeek: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Creator",
+                value: "creator"
+              }
+            ],
+            recordsInCuration: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Record maintainer(s)",
+                value: "recordMaintainers"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              }
+            ],
+            recordsWithoutDois: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Date last edit",
+                value: "updatedAt"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              },
+              {
+                text: "Last edited by",
+                value: "lastEditor"
+              }
+            ],
+            hiddenRecords: [
+              {
+                text: "Record name (id)",
+                value: "recordNameID"
+              },
+              {
+                text: "Date created",
+                value: "createdAt"
+              },
+              {
+                text: "Under curation by",
+                value: "curator"
+              },
+              {
+                text: "Created by",
+                value: "creator"
+              }
+            ]
+          },
+          search1: "",
```

Comment:
These variables should be nested into a seaches attribute and be renamed to represent the panel they belong to.

```suggestion
          searches: {
        recordsAwaitingApproval: "",
        pendingMaintenanceRequests: ""
        ...
    },,
```